jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/SelectedUnitsTab.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Fragment, FunctionComponent } from 'react';
```

Comment:
Implemented the SlectedUnits tab which shows the SortingUnitView plugins for the selected units. Maybe we want a better name than "Selected Units"